src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesDataController.java

```diff
@@ -109,4 +112,37 @@ private ResponseEntity<OverseasEntityDataApi> getOverseasEntityDataResponse(
         
         return ResponseEntity.ok(overseasEntityDataApi);
     }
+
+    @GetMapping("/beneficial-owners")
+    public ResponseEntity<PrivateBoDataListApi> getOverseasEntityBeneficialOwners(
+            @PathVariable(TRANSACTION_ID_KEY) String transactionId,
+            @PathVariable(OVERSEAS_ENTITY_ID_KEY) String overseasEntityId,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) throws ServiceException {
+
+        final var logMap = new HashMap<String, Object>();
+        logMap.put(OVERSEAS_ENTITY_ID_KEY, overseasEntityId);
+        logMap.put(TRANSACTION_ID_KEY, transactionId);
+        ApiLogger.infoContext(requestId, "Calling service to retrieve private beneficial owner information", logMap);
+
+        isRoeUpdateFlagEnabled();
+        final Optional<OverseasEntitySubmissionDto> overseasEntitySubmissionDto = overseasEntitiesService.getOverseasEntitySubmission(overseasEntityId);
+
+        if(overseasEntitySubmissionDto.isPresent() && overseasEntitySubmissionDto.get().isForUpdate()) {
+            String entityNumber = overseasEntitySubmissionDto.get().getEntityNumber();
+            PrivateBoDataListApi privateBeneficialOwnersData =  null;
```

Comment:
no need to assign null. Merge with next line.

Reply:
Fixed